ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -150,12 +150,17 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
         echo -e "[DEBUG] The group account 'video' includes the below user accounts:"
         cat /etc/group | grep video
 
+        # Change /dev/device user to www-data from root
```

Comment:
For security, please consider more safe method. 

For example, 
1) server guide: admin set /etc/group
2) then, www-data run insmod command to load kernel modules.